posthog/api/organization.py

```diff
@@ -42,10 +44,15 @@ class Meta:
 
     def create(self, validated_data: Dict, *args: Any, **kwargs: Any) -> Organization:
         serializers.raise_errors_on_nested_writes("create", self, validated_data)
-        request = self.context["request"]
-        organization, _, _ = Organization.objects.bootstrap(request.user, **validated_data)
+        organization, _, _ = Organization.objects.bootstrap(self.context["request"].user, **validated_data)
         return organization
 
+    def get_membership_level(self, organization: Organization) -> Optional[OrganizationMembership.Level]:
+        membership = OrganizationMembership.objects.filter(
+            organization=organization, user=self.context["request"].user
```

Comment:
Rather than using request here let's pass user as parameter - this way this can be tested properly?

Reply:
Not sure what you meant, since this is a serializer method fieldd?